packages/flutter/lib/src/material/slider.dart

```diff
@@ -782,7 +782,9 @@ class _SliderState extends State<Slider> with TickerProviderStateMixin {
   Widget _buildMaterialSlider(BuildContext context) {
```

Comment:
Should the docs on `Slider` be updated to reflect all of this new new? :)

Reply:
Bumping this, do we think there are any doc updates with this change?

Reply:
So sorry, i wanted to comment. Yes, I want to write this Slider doc along side the migration guide so i can mention the new stuff things in the migration guide and sync them with Slider docs.   Migration guide will have steps to restore the previous Slider components

Now that PR is approved. I will initiate migration guide draft.